closed/adds/jdk/src/solaris/native/jdk/crypto/jniprovider/NativeCrypto_md.c

```diff
@@ -67,6 +73,16 @@ void * load_crypto_library() {
         result = dlopen (libName,  flags);
     }
 
+#if defined(UNIX)
+    if (traceEnabled && (NULL != result)) {
+        struct link_map *map = NULL;
+        dlinfo(result, RTLD_DI_LINKMAP, &map);
+        fprintf(stderr, "Attempt to load OpenSSL %s\n", map->l_name);
+        fflush(stderr);
+    }
+
+#endif /* defined(UNIX) */
```

Comment:
This is all dead (because `UNIX` won't be defined); I think we should just remove it.

Reply:
Shouldn't we find the right define to use and keep the debug code?

Reply:
If nothing else `#if !defined(_AIX) && !defined(MACOSX)`

Reply:
I see `defined(__linux__)` used elsewhere.

Reply:
It can stay if we change `UNIX` to `__linux__`.

Reply:
Changed.